lms/djangoapps/courseware/views/views.py

```diff
@@ -786,6 +782,10 @@ def course_about(request, course_id):
         # Overview
         overview = CourseOverview.get_from_id(course.id)
 
+        # Embed the course reviews tool
+        from openedx.features.course_experience.views.course_reviews import CourseReviewsModuleFragmentView
```

Comment:
I'm wondering if we should render this by name, rather than having to import the class. I've temporarily forgotten how to do this, or even if it is possible. You can look the URL up by name, but that doesn't help to get the rendered fragment back. You could load the class via reflection but I'm not sure that that's any cleaner.

Just thinking out loud... probably this should stay as it is. I just don't love the coupling of this file to the course experience feature.